client/lib/followers/README.md

```diff
@@ -30,52 +30,53 @@ Fetches followers in batches of 100 starting from the given page, which defaults
 /**
```

Comment:
Most of my feedback from `client/lib/email-followers/README.md` is valid here too